programs/mmm/src/instructions/vanilla/sol_fulfill_buy.rs

```diff
@@ -323,7 +358,37 @@ pub fn handler<'info>(
     )?;
     try_close_sell_state(sell_state, payer.to_account_info())?;
 
+    // return the remaining per pool escrow balance to the shared escrow account
+    if pool.using_shared_escrow() {
+        let min_rent = Rent::get()?.minimum_balance(0);
```

Comment:
is there any case that we want to close the pool here, perhaps when the buy limit has been reached and there are no nfts left in the pool? seems like `try_close_pool` will skip this case